src/python/pants/option/config_test.py

```diff
@@ -16,8 +17,7 @@
 class ConfigFile:
     content: str
     unexpanded_default_values: Dict
-    expanded_default_values: Dict
-    expected_options: Dict
+    expected_options: Dict[str, Dict[str, list[str]]]
```

Comment:
`expected_options` now directly includes expected DEFAULT values, since we no longer inject into them, so we don't need `expanded_default_values`